packages/react-dom/src/client/ReactDOMFiberTextarea.js

```diff
@@ -107,7 +109,7 @@ export function initWrapperState(element: Element, props: Object) {
         children = children[0];
       }
 
-      defaultValue = '' + children;
+      defaultValue = getSafeValue(children);
```

Comment:
`defaultValue` was a string before and is a SafeValue now. I'm not sure if this causes issues with the equality test below.

Reply:
Should we stringify here as well, just to be safe?